R/CohortConstruction.R

```diff
@@ -253,7 +257,15 @@ generateCohort <- function(cohortId = NULL,
     rlang::inform(paste0(i, "/", nrow(cohortDefinitionSet), "- Generating cohort: ", cohortName, " (id = ", cohortId, ")"))
     sql <- cohortDefinitionSet$sql[i]
 
-    if (!isSubset) {
+    if (isSubset) {
+      sql <- SqlRender::render(
+        sql = sql,
+        cdm_database_schema = cdmDatabaseSchema,
+        cohort_table = cohortTableNames$cohortTable,
+        cohort_database_schema = cohortDatabaseSchema,
+        warnOnMissingParameters = FALSE
+      )
+    } else {  # combined cohorts apply same paramaters as standard cohort generation
```

Comment:
Same sort of re-organization here:   before, the assumption was if it's not a subset, then it must be a standard cohort generation (with sql).  But now that there's another choice, it is better to condition on positive identification `if (isSubset)`  vs. `if(!isSubset)`.